### PR TITLE
Foobar feature

### PR DESCRIPTION
**Pull request by milenkica**
Ready bruh

### CODE REVIEW

**Comment by milenko-trz** [comment]
hmm. change that.

**Comment by milenkica** [comment]
done

**Review by milenkica — COMMENTED** [review]
done2

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,8 @@ The MIT License (MIT)
 
 Copyright © 2021 
 
-dev2
+fuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuu
```
Comment (milenkica) [inline LICENSE:5]: comment

**Pull request merged**